### DISCUSSION TITLE
Sudden errors / failure to import?

### DISCUSSION THREAD

**Discussion by DruCart3456** [BadDogSkyrim/PyNifly#217] [General]
Apologies for putting in Discussions. I don't think its an issue with Nifly itself, more than smth on my side, so I thought it'd be better here. That, and, I'm just not used to using Github a lot.

I've been using 3.6 and updated to the most up to date version of Nifly a few days ago to import some 76 models. Worked perfectly, despite giving a popup that the addon was made for 4.0, up until just yesterday. 

Now it gives me errors consistently, and fails to import every time. Here, I'm importing a FO4 mod, and this is the error that pops up.
![image](https://github.com/BadDogSkyrim/PyNifly/assets/111559535/47e5fe67-c434-4965-a984-57a616a54779)
Any idea what it could be? Should I downgrade my Nifly? Should I downgrade my Blender from 3.6.12?

**Comment by DruCart3456** [comment]
Actually trying it: Downgrading to Nifly V14.4 seemed to have fixed it completely. I'm not at all sure _why_ but it at least fix my problem, though its strange it worked for days before, then suddenly failed later on. It'd make sense, seeing its made for Blender 4.1, if it had failed immediately, so I'm not sure why that was.

I'm still not sure if its a Nifly issue, though, and worth putting in Issues, or smth entirely on my side. Just strangeness.

**Comment by BadDogSkyrim** [comment]
I think this is fixed with the latest. There were a few 4.0 features that weren't properly protected from messing up 3.x.
